### PR TITLE
[fix] Fixed "Esc" key close bug in the visualizer

### LINKED ISSUE #185: [bug] Esc bug in visualizer
How to replicate:

- open topology visualizer in admin
- press ESC on keyboard
- open visualizer again
- close

Expected result: graph is shown normally also the second time and overlay is closed in the end.

Actual results: graph is not shown and JS errors are thrown.

### PR DESCRIPTION
**Pull request by Aryamanz29**
**Blockers**

- [x] https://github.com/openwisp/openwisp-network-topology/pull/195

Fixes #185

### CODE REVIEW

File: openwisp_network_topology/tests/test_selenium.py [hunk 1]
```diff
@@ -151,3 +153,38 @@ def test_topology_non_admin_view_graph_visualizer(self):
         )
         topology_graph_element.click()
         self._assert_topology_graph()
+
+    def test_topology_admin_visualizer_multiple_close_btn_append(self):
```
Comment (nemesisdesign) [inline openwisp_network_topology/tests/test_selenium.py:157]: is this test related to this PR or another one?

Comment (Aryamanz29) [reply openwisp_network_topology/tests/test_selenium.py]: This is related to the `"closeBtn"` PR. I have rebased this branch on the latest master. :+1:

**Review by nemesisdesign — APPROVED** [review]
